invisible_cities/core/core_functions_test.py

```diff
@@ -111,6 +113,37 @@ def test_in_range_right_close_interval(data):
     assert all(output)
 
 
+@given(random_length_float_arrays(min_length = 1,
+                                  min_value  = 0,
+                                  max_value  = 100))
+def test_all_in_range_when_all_fall_inside(data):
+    assert core.all_in_range(data, 0, 100, left_closed=True, right_closed=True)
+
+
+@given(random_length_float_arrays(min_length = 1,
+                                  min_value  = 0,
+                                  max_value  = 100))
+def test_all_in_range_when_some_fall_outside_silent(data):
+    minvalue =np.min(data)
```

Comment:
```suggestion
    minvalue = np.min(data)
```
here and in the next couple of tests.